api/lib/getJSON.js

```diff
@@ -99,7 +100,11 @@ const prepVerse = (row, includeMeta = false, liveSearch = 0) => {
     verse.pageNo = row.PageNo;
     verse.lineNo = row.LineNo;
     verse.updated = row.Updated;
-    verse.visraam = JSON.parse(row.Visraam);
+    try {
```

Comment:
interesting solution. what are the cases where `row.Visraam` wouldn't be a JSON?

Reply:
We have a case in prod (not sure how it got there) where the row is "" so JSON.parse trips over it. 

This is also possible in ceremonies.